src/actions/userManagement.js

```diff
@@ -152,3 +152,25 @@ export const updateUserFinalDayStatus = (user, status, finalDayDate) => {
     });
   };
 };
+
+export const updateUserFinalDayStatusu = (user, status, finalDayDate,isSet) => {
```

Comment:
The function name is not clear meaningful, what does the last char "u" mean here? And you already have one function `updateUserFinalDayStatus`,  consider changing this one to a more meaningful one.

Reply:
Sure, this one will revise in the revision.